run_tests.py

```diff
@@ -26,6 +26,16 @@
         TEST_RUNNER='django_nose.NoseTestSuiteRunner',
         SITE_ID=1,
         MAILER_EMAIL_BACKEND='mailer.tests.TestMailerEmailBackend',
+        TEMPLATES=[
+            {
+                'BACKEND': 'django.template.backends.django.DjangoTemplates',
+                'DIRS': [],
+                'APP_DIRS': True,
```

Comment:
is this what makes it use the `html` templates in the apps folders vs the haml?

Reply:
This is how it knows where to look for templates. It was added in Django 1.7 and required buy the time we get to Django 1.11